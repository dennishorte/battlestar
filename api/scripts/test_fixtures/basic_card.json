{
  "id": "card-id-123",
  "name": "Lightning Bolt",
  "lang": "en",
  "mana_cost": "{R}",
  "type_line": "Instant",
  "oracle_text": "Lightning Bolt deals 3 damage to any target.",
  "legalities": {
    "standard": "not_legal",
    "modern": "legal",
    "legacy": "legal",
    "vintage": "legal",
    "commander": "legal"
  },
  "image_uris": {
    "art_crop": "https://example.com/lightning_bolt.jpg",
    "normal": "https://example.com/lightning_bolt_normal.jpg"
  },
  "colors": ["R"],
  "color_identity": ["R"],
  "cmc": 1,
  "rarity": "common",
  "set": "TST",
  "collector_number": "185"
} 