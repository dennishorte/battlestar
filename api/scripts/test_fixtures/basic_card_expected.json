{
  "data": {
    "id": "card-id-123",
    "card_faces": [
      {
        "name": "Lightning Bolt",
        "mana_cost": "{R}",
        "type_line": "Instant",
        "oracle_text": "Lightning Bolt deals 3 damage to any target.",
        "image_uri": "https://example.com/lightning_bolt.jpg",
        "colors": ["R"]
      }
    ],
    "collector_number": "185",
    "color_identity": ["R"],
    "legal": ["modern", "legacy", "vintage", "commander"],
    "layout": null,
    "rarity": "common",
    "set": "TST",
    "cmc": 1,
    "digital": null,
    "name": "Lightning Bolt",
    "colors": ["R"],
    "type_line": "Instant"
  },
  "source": "scryfall"
} 